internal/db/update/schema.go

```diff
@@ -147,6 +191,14 @@ func (s *SchemaUpdate) Ensure(db *sql.DB) (int, error) {
 		return -1, err
```

Comment:
If there is an error applying the schema update, the database will shut down. The next time it starts, it will go through applying the schema updates again, and only enable foreign keys before actually opening the database. 

For that reason, I don't think we need to concern ourselves with errors here because an error means you don't even have a database.

Reply:
Fair point if nothing in MicroCluster is usable after this point onwards.